search-app/src/cmr/search/routes.clj

```diff
@@ -70,10 +71,17 @@
     mt/json
     mt/xml))
 
+(defconfig test-environment? 
```

Comment:
Our configurations should not have a ? in them since we use environment variables to override the value. `test-environment` will let us use the configuration parameter CMR_TEST_ENVIRONMENT.

Reply:
Removed ?